spec/modules/payloads_spec.rb

```diff
@@ -4174,6 +4184,17 @@
                           reference_name: 'windows/vncinject/reverse_tcp_rc4_dns'
   end
 
+  context 'windows/x64/encrypted_shell_reverse_tcp' do
+    it_should_behave_like 'payload is not cached',
+                          ancestor_reference_names: [
+                              'singles/windows/x64/encrypted_shell_reverse_tcp',
+                              'stagers/windows/x64/encrypted_reverse_tcp',
+                              'stages/windows/x64/encrypted_shell'
```

Comment:
In case we want to actually do some testing, these need to be part of a test for the staged version of this payload, `windows/x64/encrypted_shell/reverse_tcp`

Reply:
Adding...